plugins/tracker-resources/src/components/issues/RelationEditorPart.svelte

```diff
@@ -91,10 +93,15 @@
     <div class="tag-container">
       <div class="flex-grow">
         <div class="overflow-label max-w-30">
-          <Component
-            is={view.component.ObjectPresenter}
-            props={{ objectId: doc._id, _class: doc._class, value: doc, inline: true }}
-          />
+          {#if doc._class === 'task:class:KanbanTemplate'}
```

Comment:
Please use 
```
import task, { KanbanTemplate } from '@hcengineering/task'
{#if doc._class === task.class.KanbanTemplate}
... 
```

Reply:
Also, why we need to show it here? Relation's will be shown for real issues, so I suppose this need to be removed.

Reply:
Done